tests/test_cli.py

```diff
@@ -68,6 +68,32 @@ def test_flow_run_with_project_dir(capfd):
         assert len(found) == 1
 
 
+def test_flow_run_with_project_dir_and_select(capfd):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        shutil.copytree(project_dir, tmp_dir, dirs_exist_ok=True)
+        captured = _run_fal(
+            [
+                # fmt: off
+                "flow", "run",
+                "--project-dir", tmp_dir,
+                "--profiles-dir", profiles_dir,
+                "--select",
+                "test.py+"
```

Comment:
dbt [has prefixes](https://docs.getdbt.com/reference/node-selection/syntax#examples) for non-model selections

> The --select flag accepts one or more arguments. Each argument can be one of:
>
> - a package name
> - a model name
> - a fully-qualified path to a directory of models
> - a selection method (path:, tag:, config:, test_type:, test_name:)

What do you think about `--select script:test.py`. This would enable using scripts that do not have `.py` extension.

Reply:
I would be OK with the `.py` ending, [dbt does it too](https://docs.getdbt.com/docs/building-a-dbt-project/building-models)

> A model is a select statement. Models are defined in .sql files (typically in your models directory):
>
> - Each .sql file contains one model / select statement
> - The name of the file is used as the model name
> - Models can be nested in subdirectories within the models directory